src/frontend/src/pages/stock/LocationDetail.tsx

```diff
@@ -45,7 +46,7 @@ export default function Stock() {
       },
       {
         name: 'sublocations',
-        label: t`Stock Locations`,
+        label: t`Stock Locaons`,
```

Comment:
```suggestion
        label: t`Stock Locations`,
```

Reply:
Snuck in accidentally. A small string debug change I made to check what I had to reload for PUI to update. Will be gone on the next commit 😀